src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -384,4 +384,22 @@ public void setCreated_NonProduction(Date createdAt) {
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+
+    /**
+     * Checks whether the edit form of student has changed the section value.
+     * @param originalStudentAttribute
+     * @return true if section has changed from its original value. 
+     */
+    public boolean hasSectionChanged(StudentAttributes originalStudentAttribute) {
```

Comment:
I forgot to mention this earlier, but based on our naming conventions, the method should be named as `isSectionChanged`